From ee0f12d6b0f58865353fbef51b212390bcd2be91 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?R=C3=A9mi=20Lef=C3=A8vre?= <rlefevre@dmy.fr>
Date: Tue, 10 Dec 2019 00:54:38 +0100
Subject: testsuite: unset MAKEFLAGS when calling python

This is not recommended but +PYTHON does not seem to work on Alpine.
---
 testsuite/mk/test.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/testsuite/mk/test.mk b/testsuite/mk/test.mk
index 87d22b330b..5fe9adb70d 100644
--- a/testsuite/mk/test.mk
+++ b/testsuite/mk/test.mk
@@ -326,7 +326,7 @@ $(TIMEOUT_PROGRAM) :
 # communicate with the topmake.
 # See Note [Communicating options and variables to a submake]
 test: $(TIMEOUT_PROGRAM)
-	+PYTHON="$(PYTHON)" "$(PYTHON)" $(RUNTESTS) $(RUNTEST_OPTS) \
+	MAKEFLAGS= PYTHON="$(PYTHON)" "$(PYTHON)" $(RUNTESTS) $(RUNTEST_OPTS) \
 		$(patsubst %, --only=%, $(TEST)) \
 		$(patsubst %, --only=%, $(TESTS)) \
 		$(patsubst %, --way=%, $(WAY)) \
-- 
2.17.1

